ale_linters/swift/sourcekitlsp.vim

```diff
@@ -1,12 +1,25 @@
 " Author: Dan Loman <https://github.com/namolnad>
 " Description: Support for sourcekit-lsp https://github.com/apple/sourcekit-lsp
 
-call ale#Set('sourcekit_lsp_executable', 'sourcekit-lsp')
+let s:default_executable = 'sourcekit-lsp'
+
+call ale#Set('sourcekit_lsp_executable', s:default_executable)
```

Comment:
Instead of setting the variable to work around `ale#Var` raising errors when a variable is undefined, use `get(b:, 'ale_sourcekit_lsp_executable', get(g:, 'ale_sourcekit_lsp_executable', v:null))` below, only a little bit prettier, and check if the result `isnot v:null`.